articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+Run `npx @11ty/eleventy --serve` in the terminal to start a local web-server for your site that will reload on changes.
+
+Go to `http://localhost:8080/nameofhtmlfile` to view the page. It should display the head, header and footer as well as the content of the page.
+
+Congratulations, you've successfully converted your first static HTML page to use an EJS Eleventy layout file.
+
+## Adding More Front Matter
+
+Remember how we templated your website's header so that all the meta tags would be the same? Now that you've added your first piece of front-matter, we can add some more to account for unique values like different page titles.
+
+First, In your partial pages (such as `_head.ejs`), remove everything inside the content attribute of your meta tags and replace it with EJS variables like the example below:
+
+```html
+<meta property="og:title" content="<%- title %>">
+<meta property="og:description" content="<%- short_description %>">
+```
+Adding - after <% definites it as an EJS variable rather than a partial file which we did before. The values such as title and short-description are the names of the variables which we will use in the front-matter.
+
+Next, in one of your HTML pages, add the EJS variables you've just created and the content you want to add like so:
+```yaml
+---
+layout: _base-layout.ejs
+title: About Louise Findlay - Front-End Web Developer
+short_description: Louise Findlay has designed and developed over 20 static websites.
+---
+```
+Finally, check on your website and the front-matter content you've just added should be reflected in the code. If not, make sure you've run `npx @11ty/eleventy --serve` in the terminal to make your website automatically reload on changes.
+
+### Creating a Page Template (Layout Chaining)
+
+You've just created your basic layout containing the elements present on every page but what if you have a custom layout for only a select number of pages such as for products? This is where layout chaining comes in.
+
+Layout chaining is how you can combine multiple layout files. You create a secondary (or as many layers as you want) layout file which uses the base layout in the exact same way as you linked it to a HTML page.
+
+Create a file called `_(layoutname)-layout.ejs` in the `_includes` folder and add the layout front-matter to link it to your base layout (in the exact same way you did for your HTML file.)
+
+Then add all the code that is identical throughout all your template pages. For example, if every product page had a section with three images, then add the section and div tags but remove the image tags since each image is unique.  In its place, add an EJS variable tag such as `<%- product_image1 %>`. See the example below:
+
+```html
+<section id="test1" class="product">
+        <div class="flex-item">
+            <picture>
+                <%- product_image1 %>
+            </picture>
+        </div>
+</section>
+```
```

Comment:
This is confusing because you mention "a section with three images", but then your code example only has one image. Your explanation of what is taking place should account for the other two images or your code example should match the written example.

Reply:
Updated code example accordingly.